packages/components/src/snackbar/list.js

```diff
@@ -23,17 +30,39 @@ function SnackbarList( { notices, className, children, onRemove = noop } ) {
 	className = classnames( 'components-snackbar-list', className );
 	const removeNotice = ( id ) => () => onRemove( id );
 
+	const isReducedMotion = useReducedMotion();
+	const [ refMap ] = useState( () => new WeakMap() );
+	const [ cancelMap ] = useState( () => new WeakMap() );
+	const transitions = useTransition(
+		notices,
+		( notice ) => notice.id,
+		{
+			from: { opacity: 0, height: 0 },
+			enter: ( item ) => async ( next ) => await next( { opacity: 1, height: refMap.get( item ).offsetHeight } ),
+			leave: ( item ) => async ( next, cancel ) => {
+				cancelMap.set( item, cancel );
+				// await next( { life: '0%' } );
+				await next( { opacity: 0 } );
+				await next( { height: 0 } );
+			},
+			config: { tension: 300 },
+			immediate: isReducedMotion,
```

Comment:
I wonder if we should wrap around react-spring primitives and offer this by default.

Reply:
I'm not sure. Because there is not a unique hook for react spring animations so basically this means creating a custom hook for each one of the react-spring hooks.